Automattic/jetpack
# [WordAds]:  Remove suggestion to use ad widget

Addressed from 86-gh-Automattic/wordads-issues

#### Changes proposed in this Pull Request:

Remove suggestion to use ad widget, we want people using the ad block, not the ad widget, which will be deprecated at some point.

#### Does this pull request change what data or activity we track or use?


No.

#### Screenshots

Before:
![image](https://user-images.githubusercontent.com/692065/163846459-cb439e3f-0443-4237-8602-927995b44a43.png)

After:
![image](https://user-images.githubusercontent.com/692065/163846605-446506ca-c7f2-4352-9789-4a50bc895bf4.png)


#### Testing instructions:






* On Dashboard, go to menu Jetpack > Settings > Traffic tab.
* Enable the toggle button for "Enable ads and display an ad below each post".
* The message "You can place additional ads using the Ad widget. [Try it out!](https://old-marine.jurassic.ninja/wp-admin/customize.php?autofocus%5Bpanel%5D=widgets)" should not be shown.
